chaise-config-sample.js

```diff
@@ -7,5 +7,14 @@ var chaiseConfig = {
     showBadgeCounts: false,
     recordUiGridEnabled: false,
     recordUiGridExportCSVEnabled: true,
-    recordUiGridExportPDFEnabled: true
+    recordUiGridExportPDFEnabled: true,
+    tour: {
```

Comment:
Actually, the tour isn't custom. It's going to be a general site tour feature for chaise.